src/lib/AnnotationControls.scss

```diff
@@ -1,23 +1,32 @@
 .bp-AnnotationControls-group {
-    padding: 0 4px 0 8px;
+    padding-left: 4px;
     border-left: 1px solid $twos;
 
     .bp-AnnotationControls-regionBtn {
-        width: $controls-button-width;
-        height: $controls-button-width;
-        border-radius: 4px;
+        position: relative;
+        display: flex;
+        align-items: center;
+        justify-content: center;
 
         svg {
             fill: $white;
         }
 
         &.is-active {
-            background-color: $white;
-
             svg {
                 fill: $black;
             }
         }
+
+        &.is-active::before {
+            position: absolute;
+            z-index: -1;
+            width: 34px;
+            height: 32px;
```

Comment:
Can we use `$controls-button-width`?

Reply:
The design calls for these dimensions... not 32x32